Proposal-eng.md

```diff
@@ -0,0 +1,64 @@
+# Proposal - UIC HCC Computer Society
+
+Dear Sir / Madam:
+
+
+
+We are computer technology enthusiasts in UIC from different majors and different grades, who founded UIC HCC Computer Society (Unofficial) in October, 2016, in hope of spreading the knowledge of computer science and technology, conveying the sprit of open source, and sharing as well as focusing on the frontier development of computer science. Today, we wish to establish UIC HCC Computer Society officially under Innovation Center at the moment when UIC New Campus has moved smoothly and Innovation Center will be established soon. We hope to make contributions to students and staffs in UIC as well as the Industry-University Integration plan of UIC with the support of this platform.
```

Comment:
1. "co-founded" seems not correct in vocabulary, and suggest to change in to like "founded together"
2. For the content about IC, I suggest this expression "we hope to establish UIC HCCCS officially attaching to IC, which will be put in services soon since the UIC new campus have been moved smoothly."
3. “产学研” should be Industry-University-Research Collaboration, and expression could be like "to promote the I-U-R Collaboration with the support...."

Reply:
1. Fixed.
2. Will fix.
3. Will fix.